src/plugins/usage_collection/public/mocks.tsx

```diff
@@ -13,11 +13,20 @@ import { ApplicationUsageContext } from './components/track_application_view';
 
 export type Setup = jest.Mocked<UsageCollectionSetup>;
 
-export const createApplicationUsageTrackerMock = (): ApplicationUsageTracker => {
-  const applicationUsageTrackerMock: jest.Mocked<ApplicationUsageTracker> = {
-    setCurrentAppId: jest.fn(),
+// This is to avoid having to mock every private property of the class
+type ApplicationUsageTrackerPublic = Pick<ApplicationUsageTracker, keyof ApplicationUsageTracker>;
+
+export const createApplicationUsageTrackerMock = (): ApplicationUsageTrackerPublic => {
```

Comment:
If we're removing `applicationUsageTracker`, why do we still need `createApplicationUsageTrackerMock`?